viewer/src/data/model/ParsedSector.ts

```diff
@@ -3,11 +3,12 @@
  */
 
 import { LevelOfDetail } from './LevelOfDetail';
-import { Sector, SectorQuads, SectorMetadata } from '../../models/cad/types';
+import { SectorQuads, SectorMetadata, Sector } from '../../models/cad/types';
+import { ParseSectorResult, ParseCtmResult } from '../../workers/types/parser.types';
 
 export interface ParsedSector {
   id: number;
-  data: null | Sector | SectorQuads;
+  data: null | ParseSectorResult | ParseCtmResult | Sector | SectorQuads;
   levelOfDetail: LevelOfDetail;
   metadata: SectorMetadata;
 }
```

Comment:
It seems a bit odd that we pass all result types through the same interface, but I'll leave it be now